_includes/build/access_token.html

```diff
@@ -35,9 +35,9 @@ <h4>
 	<p>
 		cURL Option 1: This cURL command requires separate base-64 encryption. We have concatenated the base64 encoding of the ‘Client ID : Secret’ as the argument to the -H flag. Please note that the URL in the Production environment will be different. 
 	</p>
-<pre><code>curl -d '' -X POST 'https://sandbox.bcda.cms.gov/ ' \ 
+<pre><code>curl -d '' -X POST 'https://sandbox.bcda.cms.gov/auth/token ' \ 
```

Comment:
copying + pasting the command doesn't work for me; I get a 
```
Bad Request
curl: (3) URL using bad/illegal format or missing URL
```

but I think it's user error.

Reply:
yeah I noticed the same thing with the newline backslashes; when I put it all on one line it works. I think that would be a separate improvement to the website to change all the code commands that we're expecting people to copy-paste.

Reply:
that's fair - is it expected that the people that would be referencing these commands will have the technical know how to modify this command to make it work?

Reply:
Actually I figured it out, it's just this command and maybe another one that I had tried on this page. I removed the extra spaces at the end of the URL and after the backslashes and it seems to work correctly now if you can test it out. Thanks for double-checking this!

Reply:
nice that works!! I fiddled with the white spaces before and must have missed one. Approving!

Reply:
hmm.. I took out the white spaces at the end of the URL and after the backslash but I get 'Bad Request' -- should I be doing something else? @kyeah